src/main/java/com/spotify/autoscaler/ServiceInit.java

```diff
@@ -0,0 +1,96 @@
+/*-
+ * -\-\-
+ * bigtable-autoscaler
+ * --
+ * Copyright (C) 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.autoscaler;
+
+import com.spotify.autoscaler.client.StackdriverClient;
+import com.spotify.autoscaler.db.Database;
+import com.spotify.metrics.ffwd.FastForwardReporter;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import javax.inject.Inject;
+import org.glassfish.grizzly.http.server.HttpServer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ServiceInit {
```

Comment:
I'm not a Dagger expert, but can this class be merged with Main? Conceptually they are a bit similar?

Reply:
It is merged with Main now :+1: